app/models.py

```diff
@@ -45,6 +45,11 @@ class Account(db.Model):
     currency: Mapped[str]
     symbol: Mapped[str]
 
+    @property
+    def get_balance(self, balance = 0):
+        self.balance = balance
+        return self.balance
```

Comment:
Если делаешь property, get не нужен 🤝
```suggestion
    def balance(self):
        return self._balance
```